modules/apps/foundation/frontend-theme/frontend-theme-admin/src/css/_custom.scss

```diff
@@ -229,6 +229,15 @@ body {
 			max-width: 150px;
 		}
 	}
+
+	.color-scheme-selector-container {
+		float: left;
+		margin: 10px;
+	}
+
+	.float-end {
```

Comment:
Instead of adding an html element for clearing floats, could we use a pseudo element instead? Bootstrap provides a mixin for that See https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_clearfix.scss#L13. Also see https://css-tricks.com/snippets/css/clear-fix/.